VariantEncoding.md

```diff
@@ -386,11 +386,15 @@ The Decimal type contains a scale, but no precision. The implied precision of a
 | Exact Numeric        | decimal8                    | `9`     | DECIMAL(precision, scale)   | 1 byte scale in range [0, 38], followed by little-endian unscaled value (see decimal table)                         |
 | Exact Numeric        | decimal16                   | `10`    | DECIMAL(precision, scale)   | 1 byte scale in range [0, 38], followed by little-endian unscaled value (see decimal table)                         |
 | Date                 | date                        | `11`    | DATE                        | 4 byte little-endian                                                                                                |
-| Timestamp            | timestamp                   | `12`    | TIMESTAMP(true, MICROS)     | 8-byte little-endian                                                                                                |
-| TimestampNTZ         | timestamp without time zone | `13`    | TIMESTAMP(false, MICROS)    | 8-byte little-endian                                                                                                |
+| Timestamp            | timestamp with time zone    | `12`    | TIMESTAMP(isAdjustedToUTC=true, MICROS)     | 8-byte little-endian                                                                                                |
+| Timestamp            | timestamp without time zone | `13`    | TIMESTAMP(isAdjustedToUTC=false, MICROS)    | 8-byte little-endian                                                                                                |
```

Comment:
While we are trying to correct this it seems like timetamp without time zone should be the logical type and the physical type is int64?

Reply:
I see this duplicates the the question @RussellSpitzer asked abve.

Reply:
The question is answered in the paragraph below.  Although "behaves" the same is ambiguous.  I'm not sure if the intent here Is if parquet has flexibility to change the physical type.  We should make sure we clarify this

Reply:
Thanks. Just saw the explanation below for logical type.
I think it means: for a string, the engines can choose to encode differently but when it's read, they should be the same.

I updated the logical types for newly added types. Since engines may choose different precisions to encode, they have the same logical type from the  paragraph below.

```
| Timestamp            | timestamp with time zone    | `12`    | TIMESTAMP(isAdjustedToUTC=true, MICROS)     | 8-byte little-endian                                                                        |
| Timestamp            | timestamp with time zone   | `22`    | TIMESTAMP(isAdjustedToUTC=true, NANOS)       | 8-byte little-endian                                                                        |
```